net-wireless/uhd/uhd-4.0.0.0.ebuild

```diff
@@ -1,12 +1,11 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-#not sure why, but eapi 7 fails
+EAPI=7
```

Comment:
Thanks for handling EAPI 7 + cmake-utils -> cmake

Reply:
My pleasure, and thank you for review this ebuild!